services/operatorsinfo/operatorsinfo_inmemory.go

```diff
@@ -92,7 +102,11 @@ func (ops *OperatorsInfoServiceInMemory) startServiceInGoroutine(ctx context.Con
 			ops.logger.Error("Fatal error opening websocket subscription for new socket registrations", "err", err, "service", "OperatorPubkeysServiceInMemory")
 			panic(err)
 		}
-		ops.queryPastRegisteredOperatorEventsAndFillDb(ctx)
+		err = ops.queryPastRegisteredOperatorEventsAndFillDb(ctx)
+		if err != nil {
+			ops.logger.Error("Fatal error querying past registered operator events and filling db", "err", err, "service", "OperatorPubkeysServiceInMemory")
+			panic(err)
```

Comment:
😭 panic

Reply:
yeah.. didn't want to fix this right now... that will create a bunch more breaking changes if I return error from this function. Will do another day :)